setup.cfg

```diff
@@ -1,2 +1,11 @@
 [bdist_wheel]
 universal = 0
+
+[flake8]
+max-line-length = 88
```

Comment:
I don't think we are running `flake8` anymore. Just `black`. But it does not hurt.
I myself have a small custom script to call flake8 and ignore a few common problems:

```
$ cat ~/bin/pychecker3
#! /bin/sh
#infile = $1
if test "x$1" = "x--stdlib"; then
    # --stdlib gets passed by some python-mode.el versions.  pep8 and
    # pyflakes and flake8 cannot handle this, so we ignore this argument, by
    # shifting the rest of the arguments.
    #shift
    # Actually, we suddenly get passed 'pychecker --stdlib /Volumes/Macintosh /Volumes/Macintosh /Volumes/Macintosh /Volumes/Macintosh /Volumes/Macintosh /Users/mauritsvanrees/myfile.py'
    # So we shift to the last argument and pass only that
    shift $(($# - 1))
fi
# Check with Python 3.
# E203 Whitespace before ':' (false positives when using black)
# E231 missing whitespace after ',' (conflicts with black)
# E501 line too long
# W503 Line break occurred before a binary operator [outdated]
flake8 --ignore=E203,E231,E501,W503 $1 | sort-flake8
```

And then `sort-flake8`:

```
#!/usr/bin/env python
# Example: flake8 zest/releaser/utils.py | ./pychecker-sort
# Sorts with Failures first.
import sys
lines = sys.stdin.read().splitlines()


def get_code(line):
    """Extract error code from line.

    Example line:
    'zest/releaser/utils.py:797:1: E302 expected 2 blank lines, found 1'
    """
    if line.count(':') != 3:
        return ''
    filename, lineno, pos, error = line.split(':')
    # Example error:
    # ' E302 expected 2 blank lines, found 1'
    code = error.strip().split(' ', 1)[0]
    if not code:
        return ''
    return code


def sortable_code(code):
    """Make code sortable.

    First failures, then errors, then warnings, then any others.
    """
    if not code:
        # sort last
        return '9'
    first = code[0]
    if first == 'F':
        value = 0
    elif first == 'E':
        value = 1
    elif first == 'W':
        value = 2
    else:
        value = 3
    return '{}{}'.format(value, code)


def error_value(line):
    return sortable_code(get_code(line))


for line in sorted(lines, key=error_value):
    print(line)
```